lib/views/file-patch-view.js

```diff
@@ -76,7 +76,7 @@ export default class FilePatchView extends React.Component {
   renderEmptyDiffMessage() {
     return (
       <div className="is-blank">
-        <span className="icon icon-info">File has no contents</span>
+        <span className="icon icon-info">File has no changed contents</span>
```

Comment:
How do you feel about "No changes to display" or similar? It also works well if and when we decide to handle ignoring whitespace.